app/src/main/res/values/strings.xml

```diff
@@ -174,6 +174,7 @@
   <string name="prefs_highlight_bookmarks_summary">Highlight bookmarked ayahs while reading</string>
   <string name="prefs_translation_text_title">Translation text size</string>
   <string name="prefs_translations">Translations</string>
+  <string name="prefs_more_translations">More Translations</string>
```

Comment:
`prefs_translations` is used in the Settings, `prefs_more_translations` is not used and not needed. Please remove it.